proto/api/v2/scan_component.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package v2;
+
+import "api/v2/vulnerability.proto";
+
+option go_package = "./api/v2;v2";
+option java_package = "io.stackrox.proto.api.v2";
+
+message ScanComponent {
+  string name = 1;
+  string version = 2;
+  License license = 3;
+  repeated EmbeddedVulnerability vulns = 4;
+  SourceType source = 5;
+  string location = 6;
+  oneof set_top_cvss {
+    float top_cvss = 7;
+  }
+  float risk_score = 8;
+  // Component version that fixes all the fixable vulnerabilities in this component.
+  string fixed_by = 9;
+  message Executable {
+    string path = 1;
+    repeated string dependencies = 2;
+  }
+  // Values are cleared after moving to cache, remove them from the grpc return as well
+  repeated Executable executables = 10;
```

Comment:
I'm guessing this is copied over from existing protos.  Need to check (or at TODO to check) with scanner to see if they still give us this and remove it if not.